packages/mux-uploader-react/README.md

```diff
@@ -0,0 +1,91 @@
+      <MuxUploader url="authenticated-url">
+        <MuxUploaderDrop slot="dropzone" text="Upload to stream.new" fullscreen overlay></MuxUploaderDrop>
+      </MuxUploader>
+    </div>
+  );
+};
+```
+
+### Attributes
+
+#### `MuxUploader`
+
+| Attribute          | Type      | Description                                                                                                                                                                                                               | Default     |
+| ------------------ | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| `url`              | `string`  | The authenticated URL that your file will be uploaded to. Check out the [direct uploads docs](https://docs.mux.com/guides/video/upload-files-directly#1-create-an-authenticated-mux-url) for how to create one. Required. | `undefined` |
+| `id`               | `string`  | An ID that allows `MuxUploaderDrop` to locate `MuxUploader`. Not necessary unless the unlikely scenario you need to nest `MuxUploader` inside `MuxUploaderDrop`.                                                          | N/A         |
+| `type`             | `"bar"`   | Specifies the visual type of progress bar. A radial type is in-progress.                                                                                                                                                  | "bar"       |
+| `uploadInProgress` | `boolean` | Toggles visual status of progress bar while upload is in progress.                                                                                                                                                        | false       |
+| `uploadError`      | `boolean` | Toggles visual status of progress bar when upload encounters an error.                                                                                                                                                    | false       |
+| `status`           | `boolean` | Toggles text status visibility of progress bar. The text that is displayed is a percentage by default. If you prefer just the progress bar with no text upload status, don't include this attribute.                      | false       |
+
+#### `MuxUploaderDrop`
+
+| Attribute      | Type      | Description                                            | Default |
+| -------------- | --------- | ------------------------------------------------------ | ------- |
+| `fullscreen`   | `boolean` | Toggles fullscreen drag and drop (work-in-progress).   | false   |
+| `overlay`      | `boolean` | Toggles fullscreen overlay on dragover.                | false   |
+| `disableDrop ` | `boolean` | Toggles off drag and drop which is enabled by default. | false   |
+
+### Methods
```

Comment:
suggestion(blocking): The organization of this information is confusing. We should probably group methods/attributes/other details per-component, or at least call out that this method is a method for `MuxUploader`